Mathlib/Algebra/Module/End.lean

```diff
@@ -42,6 +42,7 @@ def Module.toAddMonoidEnd : R →+* AddMonoid.End M :=
     -- Somehow, now that `SMul` is heterogeneous, it can't unfold earlier fields of a definition for
     -- use in later fields.  See
     -- https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/Heterogeneous.20scalar.20multiplication
+    -- jmc: there should be a rw-lemma `smul_comp` close to `SMulZeroClass.compFun`
```

Comment:
Can you convert this to a TODO or FIXME (or implement this), please? I think "jmc" comments might fall through the cracks.